src/storage/hummock_sdk/src/table_watermark.rs

```diff
@@ -19,11 +19,14 @@ use std::mem::size_of;
 use std::ops::Bound::{Excluded, Included, Unbounded};
 use std::sync::Arc;
 
+use bincode::{Decode, Encode};
```

Comment:
We need this because we define `Decode` `Encode` on `WatermarkSerdeType`. Is this neccessary?

Reply:
I think the reason why we need it is because we need it for hummock trace. Like other trace related struct, let's not leak encode/decode outside of hummock trace and handle it [here](https://github.com/risingwavelabs/risingwave/blob/3500061180b4eafb23dbeed1c41c0ee7d00ae97d/src/storage/src/store.rs#L831) because we can just use a bool to do the encode and decode in hummock trace.